docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -1,56 +1,97 @@
+
+    For example, list all data sets under your user ID by entering your information in the following example command:
+    ```
+    zowe zos-files list data-set "IBMUSER.*"
+    ```
+
+    A list of data sets is returned, indicating Zowe CLI is successfully configured to access a z/OSMF instance.
+
+    If the CLI returns an error message, verify that you have access to the target system. Examine the configuration files in a text editor to check that the entered information is correct.
+
+**Important:** After the configuration file is in place (by using either the `zowe config init` command or a file provided by a system administrator), the `zowe profiles` commands used in Zowe v1 no longer function. Zowe CLI returns errors when deprecated profile commands are issued.
+
+## Creating team plug-in profiles
+
+After the `zowe.config.json` team configuration file is created and new plug-ins installed, run the `zowe config init` (or `zowe config auto-init`, if using the API ML) command again to add the plug-in profiles to the configuration file.
+
+1. Install a new plug-in.
+
+    For example, run the following command to install the [IBM CICS Plug-in](../user-guide/cli-cicsplugin.md):
 
-    **Example:** List all data sets under your user ID:
     ```
-    zowe zos-files list data-set "MY.DATASET.*"
+    zowe plugins install @zowe/cics-for-zowe-cli
     ```
 
-    A list of data sets is returned. You successfully configured Zowe CLI to access a z/OSMF instance.
+    **Note:** If the `zowe.config.json` file has not yet been created in the `.zowe` directory, see [Creating team profile configuration files](#creating-team-profile-configuration-files).
+
+2. Run the `zowe config init --global-config` or `zowe config auto-init --global-config` command.
+
+    This adds a plug-in profile to the configuration file in the `.zowe` root directory.
+
+3. Open the `zowe.config.json` file and confirm the plug-in profile is included.
     
-    If the CLI returns an error message, verify that you have access to the target system. Examine your configuration files in a text editor to verify that the information you entered is correct.
+    In the example from Step 1, the profile information displays similarly to the example below:
 
+    ```
+    "cics": {
+            "type": "cics",
+            "properties": {
+                "port": 123
+            },
+            "secure": []
+        }
+    ```
+
+    The plug-in profile has been successfully added to the `zowe.config.json` file in the `.zowe` root directory.
 
-**Important:** After the configuration files are in place (either by using the `zowe config init` command or by manually creating the files), the now-deprecated zowe profiles commands no longer function. Zowe CLI returns errors when you attempt to use deprecated profile commands.
+    **Note:** To add plug-in profiles to a configuration file in the current working directory, enter the base command without the option: `zowe config init`.
```

Comment:
What do you think of specifying the option name here:
_without the`--global-config` option_

Reply:
At first I thought adding the option code would make things hard to read, but I've now changed my mind. Updated to add the option code.